etc/org.opencastproject.transcription.microsoftazure.MicrosoftAzureTranscriptionService.cfg

```
@@ -0,0 +1,59 @@
+# Change enabled to true to enable this service.
+enabled=false
+
+# Microsoft Azure Service details
+subscription.key=
+region=
+
+# Language code of the supplied audio. See the Google Speech-to-Text service documentation
+# for available languages. If empty, the default will be used ("en-US").
+# List of supported languages: https://cloud.google.com/speech-to-text/docs/languages
+#language=en-US
+
+# Filter out profanities from result. Valid values: raw, remove, mask
+# Raw: Includes the profanity in the result.
+# Remove: Removes all profanity from the result.
+# Mask: Replaces profanity with asterisks.
+#profanity.option=Raw
+
+# Default is VTT. Set this to true if you want SRT instead.
+#use.subrip.format=false
+
+# A phrase list is a list of words or phrases provided ahead of time to help improve their recognition. Adding a phrase
```

Very cool.  How many can I add?  Any defined maximum lengths for individual phrases, or the total set of phrases? :D

---

Maximum of 500 phrases. Technically there's not max length for individual phrases (except what's imposed by java.lang.String), but I'm hoping the description and examples make it clear that you're not supposed to write down long paragraphs of phrases :D

---

I'd be curious what the failure mode there is.  Does it just silently drop 501 and higher?  Does it blow up?

I'm (relatively) sure that this won't bite us, but I can see a particularly... _enthusiatic_ admin collecting a corpus of technical jargon, and then dumping the whole thing into the config file.  If that limit is documented in Microsoft's docs, maybe a link would be appropriate?